### ISSUE TITLE
Confused about why using training dataset as validation dataset?

### ISSUE THREAD

**Issue by cuttle-fish-my** [MiracleDance/PoseRAC#5]
In the file `train.py`, the function `main` reads:
```
train_landmarks, train_labels = csv2data(train_csv, action2index, num_classes)
valid_landmarks, valid_labels = csv2data(train_csv, action2index, num_classes)
```
which indicates we need to use training dataset as validation dataset.

Why not just use the validation directly?

**Comment by MiracleDance** [comment]
Oh, your suggestion is correct, the most standard practice is indeed to separate the training set and the validation set. 

Just for the convenience of debugging, the validation dataset is not used by me. But you can use it for validation, the generalization may be better, and the final accuracy rate may be higher.

But overall, even if the validation set is not used, it will not have much impact. The current model can already achieve great results in the test set.

**Comment by cuttle-fish-my** [comment]
> 

Thanks!
